docs/developer/quickstart.md

```diff
@@ -46,10 +46,10 @@ This command will build the binary and output it to the `/build/.out` directory.
 To build an NGINX Kubernetes Gateway container image from source run the following make command:
 
 ```shell
-make TAG=$(whoami) container
+make container
 ```
 
-This will build the docker image and tag it with your user ID, e.g. `docker.io/library/nginx-kubernetes-gateway:user`
+This will build the docker image `nginx-kubernetes-gateway:edge`.
```

Comment:
Hmm my image is prefixed  with `docker.io/library/...`. Is that just me?

Reply:
> docker.io/library/

it is assumed by default and can be omitted